src/code_f2470_len_27f0.c

```diff
@@ -4,7 +4,16 @@ INCLUDE_ASM(code_f2470_len_27f0, func_802CDAC0);
 
 INCLUDE_ASM(code_f2470_len_27f0, set_npc_animation);
 
-INCLUDE_ASM(code_f2470_len_27f0, DeleteNpc);
+s32 DeleteNpc(script_context* script, s32 initialCall) {
+    bytecode* ptrReadPos = script->ptrReadPos;
+    npc* npcPtr = get_npc_unsafe(get_variable(script, *ptrReadPos));
```

Comment:
Probably better to `*ptrReadPos++` for good measure, since the arg is being consumed

Reply:
Changed